mne/minimum_norm/inverse.py

```diff
@@ -746,6 +747,42 @@ def _check_ori(pick_ori, source_ori):
                            'inverse operator with fixed orientations.')
 
 
+def _check_loose_forward(loose, forward, loose_as_fixed=(0., None)):
+    """
+    XXX there an API glitch with loose that means fixed inverse between
+    sparse solvers and MNE. For sparse solvers loose=None means fixed
+    and for MNE loose = None means free orientation...
```

Comment:
We should fix this for release. Perhaps in 0.16 disallow `loose=None` and force people to be explicit about it.